CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # PyPRT ChangeLog
 
+## v1.4.0 (2021-08-XX)

```

Comment:
Please set the final release date on develop, so we can avoid additional commits to master